sources/packages/backend/apps/db-migrations/src/sql/SFASApplicationDependents/Create-sfas-application-dependents.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE sims.sfas_application_dependents (
+    id INT PRIMARY KEY,
+    application_id INT NOT NULL,
+    dependent_name VARCHAR(25),
+    dependent_birth_date DATE,
+    -- Audit columns
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+    updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+    extracted_at TIMESTAMP WITH TIME ZONE NOT NULL
+);
+
+CREATE INDEX sfas_application_dependents_application_id ON sims.sfas_application_dependents(application_id);
```

Comment:
Pretty minor but I do not follow the logic to make some words start with upper case sometimes in the middle of the sentences, for instance, `Dependents` vs `dependent` or `Last name`.

Reply:
Side effect of copy/paste from document. I can update that sticking to our standards.